pkg/cmd/server/origin/master.go

```diff
@@ -162,6 +162,12 @@ func (c *MasterConfig) newAssetServerHandler() (http.Handler, error) {
 	if err != nil {
 		return nil, err
 	}
+	backend, policy, err := kubernetes.GetAuditConfig(c.Options.AuditConfig)
```

Comment:
This method is called from a method with access to the genericapiserver config.  Pass that config through to here and assign it.  We probably want all apiservers to use the same audit config and that is the source of truth at this level.